chart/values.yaml

```diff
@@ -315,6 +324,11 @@ controller:
   # Timeout in seconds for the pre-delete hook
   preDeleteHookTimeoutSeconds: 120
 
+# Configures the Pod Security Context
+# https://kubernetes.io/docs/tasks/configure-pod-container/security-context
+podSecurityContext:
+  runAsNonRoot: true
```

Comment:
I think this should probably go under `controller`, unless I'm missing a specific reason to have it as a global?